src/pages/postgraphile/smart-tags.md

```diff
@@ -587,6 +587,8 @@ the following syntax which mirrors the PostgreSQL foreign key constraint:
 
 `@foreignKey (col1, ...) references [my_schema.]my_table [(col1, ...)]`
 
+In the tags file you have to omit the leading `@foreignKey` (see example below).
```

Comment:
```suggestion
In the tags file you must omit the leading `@foreignKey ` text since it is specified as the key (rather than the value) in the tags object ─ see example below.
```